superduperdb/db/base/download.py

```diff
@@ -10,10 +10,10 @@
 from multiprocessing.pool import ThreadPool
 
 import boto3
-import requests
 from tqdm import tqdm
 
 from superduperdb import CFG, logging
+from superduperdb.misc import requests
```

Comment:
To me, we shouldn't be adding this into the core package. This is a feature we only require for testing.

If we truly want to have functionality for reading a local filesystem, I would use a dedicated package for this such as `fsspec`.